cli/src/diff_util.rs

```diff
@@ -176,6 +184,7 @@ fn default_diff_format(
     match name.as_ref() {
         "summary" => Ok(DiffFormat::Summary),
         "types" => Ok(DiffFormat::Types),
+        "paths-only" => Ok(DiffFormat::NameOnly),
```

Comment:
Same here: `name-only`

I'm guessing you went back and forth on the naming. I can see `--name-only` being mistaken to mean "filename only, no path" (which is probably what you were also thinking?),  but it's probably not a big risk. So I think I agree with just copying git's `--name-only`.